authorizations/authorization-common/src/main/java/org/apache/gravitino/authorization/common/AuthorizationProperties.java

```diff
@@ -34,21 +34,4 @@ public AuthorizationProperties(Map<String, String> properties) {
   public abstract String getPropertiesPrefix();
 
   public abstract void validate();
-
-  public static void validate(String type, Map<String, String> properties) {
```

Comment:
Are we going to remove this method directly?

Reply:
Yes, this method is only used by chainedAuthzPlugin.

Reply:
I see.